src/MIDebugEngine/AD7.Impl/AD7Property.cs

```diff
@@ -71,11 +71,14 @@ public DEBUG_PROPERTY_INFO ConstructDebugPropertyInfo(enum_DEBUGPROP_INFO_FLAGS
 
             if ((dwFields & enum_DEBUGPROP_INFO_FLAGS.DEBUGPROP_INFO_ATTRIB) != 0)
             {
-                // don't check readonly attribute, it doubles the eval time for a variable
-                //if (this.m_variableInformation.IsReadOnly())
-                //{
-                //    propertyInfo.dwAttrib |= enum_DBG_ATTRIB_FLAGS.DBG_ATTRIB_VALUE_READONLY;
-                //}
+                // Only check this property if we're running under clrdbg, other engines double the eval time.
```

Comment:
Is it fast for clrdbg because we have a fast implementation of VarShowAttributes?